python/llm/dev/benchmark/all-in-one/run-arc.sh

```diff
@@ -1,5 +1,9 @@
 source /opt/intel/oneapi/setvars.sh
+
 export USE_XETLA=OFF
-export SYCL_PI_LEVEL_ZERO_USE_IMMEDIATE_COMMANDLISTS=1
+KERNEL_VERSION=$(uname -r)
+if [[ $KERNEL_VERSION != *"6.5"* ]]; then
```

Comment:
what if 6.5 and higher?

Reply:
we don't set `SYCL_PI_LEVEL_ZERO_USE_IMMEDIATE_COMMANDLISTS` for 6.5 or higher versions, namely do nothing.